_data/socialmedia.yml

```diff
@@ -7,45 +8,8 @@
   url: https://twitter.com/hpccarpentry
   class: fa-twitter
   title: ""
-  
-# - name: Facebook
-#   url: http://www.facebook.com/carpentries
-#   class: fa-facebook
-#   title: ""
 
 - name: Slack
   url: https://swc-slack-invite.herokuapp.com/
-  class: fa-slack 
+  class: fa-slack
   title: ""
-  
-# - name: YouTube
```

Comment:
I don't think any of these do any harm and leave valuable breadcrumbs for our future selves

Reply:
Rule 7. Look at the upstream file for ideas instead of carrying around the weight ourselves.

Reply:
Again, this is trivialising the effort involved in doing that. Why would you even believe that they exist upstream if there is no indication of that in these files?

Reply:
Restored by 295ab288. Haven't heard of a few of these.